src/Microsoft.Health.Fhir.Core/Resources.resx

```diff
@@ -312,6 +312,9 @@
   <data name="SearchParameterDefinitionInvalidResource" xml:space="preserve">
     <value>bundle.entry[{0}].resource is not a SearchParameter resource.</value>
   </data>
+  <data name="SearchParameterKeyRequired" xml:space="preserve">
+    <value>For search parameters key is required</value>
```

Comment:
How about "Name is required for search parameter."?

Reply:
Removed the error message.